public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -238,10 +238,10 @@ export const RulesGroup: FC<Props> = React.memo(({ group, namespace, expandAll,
       )}
       {isEditingGroup && (
         <EditCloudGroupModal
-          groupInterval={group.interval ?? ''}
-          nameSpaceAndGroup={{ group: group, namespace: namespace }}
-          sourceName={getRulesSourceName(namespace.rulesSource)}
+          namespace={namespace}
+          group={group}
           onClose={() => closeEditModal()}
+          folderUrl={folder?.canEdit ? makeFolderSettingsLink(folder) : undefined}
```

Comment:
blocking: When testing this it seems like the `folderUrl` is empty when editing an evaluation group from the alert creation flow – is there anything more we need to do to have the correct folder URL?

Reply:
The folder link should only be available when you edit a group from the alert list view.
Evaluation group editing on the alert creation flow should only allow editing the interval, so the missing folder link is a deliberate thing there (we left eval group editing in this place as fallback compatibility. I think we need to think about eval group experience in the revamped alert creation flow)